jina/executors/indexers/vector.py

```diff
@@ -115,6 +117,7 @@ def add(self, keys: 'np.ndarray', vectors: 'np.ndarray', *args, **kwargs) -> Non
         self._validate_key_vector_shapes(keys, vectors)
         self.write_handler.write(vectors.tobytes())
         self.key_bytes += keys.tobytes()
+        self.existing_keys = np.concatenate((self.existing_keys, np.repeat(True, len(keys))))
```

Comment:
You can use `np.full()` instead of `np.repeat`. That is the recommended way, as far as I understand here https://stackoverflow.com/questions/21174961/how-to-create-a-numpy-array-of-all-true-or-all-false

Reply:
Oh yeah, I was actually using that in other places. Should make it consistent